arm-web/2016-06-01/connectionGateways.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
please fix path, the path to the spec should contain "swagger" folder after "api-version" folder